gstd/src/msg/encoded.rs

```diff
@@ -16,7 +16,16 @@
 // You should have received a copy of the GNU General Public License
 // along with this program. If not, see <https://www.gnu.org/licenses/>.
 
-//! Module with encoded via scale-codec messaging functions.
+//! Module with messaging functions (load, reply, send) for operating 
+//! with messages arguments as with data structure instead of bytes array 
+//! decoded/encoded via SCALE Codec.
```

Comment:
Should we ref parity-scale-codec lib?

Reply:
I suppose no, it is in the 1st place in google results.